docs/core/tools/telemetry.md

```diff
@@ -14,20 +14,20 @@ ms.assetid: 480df976-7568-4df4-9d26-9911357b5a31
 
 # .NET Core CLI Tools telemetry
 
-The .NET Core CLI Tools include a [telemetry feature](https://github.com/dotnet/cli/pull/2145) that collects usage information. It's important that the .NET Team understands how the tools are used so that we can improve them. For more information, see [What we've learned from .NET Core SDK Telemetry](https://blogs.msdn.microsoft.com/dotnet/2017/07/21/what-weve-learned-from-net-core-sdk-telemetry/).
+The [.NET Core Command-Line Interface (CLI) Tools](index.md) includes a [telemetry feature](https://github.com/dotnet/cli/pull/2145) that collects usage information. It's important that the .NET Team understands how the tools are used so that we can improve them. For more information, see [What we've learned from .NET Core SDK Telemetry](https://blogs.msdn.microsoft.com/dotnet/2017/07/21/what-weve-learned-from-net-core-sdk-telemetry/).
 
 The collected data is anonymous and published in an aggregated form for use by both Microsoft and the community under the [Creative Commons Attribution License](https://creativecommons.org/licenses/by/4.0/). 
 
 ## Scope
 
-The `dotnet` command is used to launch both apps and the .NET Core CLI Tools. The `dotnet` command itself doesn't collect telemetry. The .NET Core CLI Tools commands run by the `dotnet` command collect the telemetry.
+The `dotnet` command is used to launch both apps and the .NET Core CLI. The `dotnet` command itself doesn't collect telemetry. The .NET Core CLI commands run by the `dotnet` command collect the telemetry.
 
-Telemetry *isn't enabled* when using the .NET Core commands:
+Telemetry *isn't enabled* when using the `dotnet` command itself, with no command attached:
```

Comment:
Prob a comma-splice before "with," but I suggest the word "itself" could be dropped and "without" used.

> Telemetry \*isn't enabled* when using the \`dotnet` command without a command attached: